lib/challenge.js

```diff
@@ -230,10 +253,19 @@ let nums = [
 
 function prompt11() {
   let median;
-  // YOUR CODE HERE
+  if (nums.length % 2 === 0) {
+    console.log("even");
+    median = (num[num.length / 2-1] + num[num.length /2]) /2;
+  } else {
+    console.log("odd")
+    median = num [Math.floor(num.length / 2)];
+  }
 
   console.log(median);
 }
+
+// Got confused on what went around the Math.floor 
+
```

Comment:
So close! You forgot to sort the array first 😎  
Nice job on the odd and even logic